From d9cf99ad7ea14d1c17c2d034d89d97854e889643 Mon Sep 17 00:00:00 2001
From: Jon Anderson <jon.anderson@joyent.com>
Date: Tue, 12 Mar 2019 14:52:05 -0400
Subject: [PATCH] MANTA-4165 Remove diffutils package pre-requisite from Muppet
 readme Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com> Approved
 by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

---
 README.md | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/README.md b/README.md
index 2ca0b1e..e31fb65 100644
--- a/README.md
+++ b/README.md
@@ -28,16 +28,14 @@ Run `make prepush` before commits; otherwise, follow the
 
 ## Prerequisites
 
-To properly run the muppet tests, two additional packages need to be installed
-from pkgsrc.
+To properly run the muppet tests, the following package must be installed from
+pkgsrc:
 
-  -  haproxy
-  -  diffutils
+- haproxy
 
 Though the version of haproxy might differ from the running `loadbalancer` zone,
 it is simply used to check that haproxy can properly parse the resulting
-`haproxy.cfg` files generated in the tests.  Additionally, `GNU diff` is used to
-compare the generated files with an expected output.
+`haproxy.cfg` files generated in the tests.
 
 ## Running the Tests
 
-- 
2.21.0

